contrib/manual_tests/test.sh

```diff
@@ -297,6 +336,64 @@ function test::manual_start_tls() {
   for i in $(seq "$n_alphas"); do
     dgraph::healthcheck_alpha_tls "$i"
   done
+
+  local count
+  for i in $(seq "$n_alphas"); do
+    count="$(dgraph::increment "$i" --tls_cacert "$TLS_PATH"/ca.crt)"
+    if [ "$i" -ne "$count" ]; then
+      log::error "Expected increment: $i but got: $count"
+      return 1
+    fi
+  done
+}
+
+# Test manual start with both internal and external TLS enabled.
+function test::manual_start_tls2() {
+  dgraph::generate_tls
+
+  local -r n_zeros=2
```

Comment:
This is an odd configuration. Why 2 nodes? The nodes should always be in odd number, 3 or 5 so that they can form a quorum.